src/tests/models/manager.js

```diff
@@ -33,6 +47,42 @@ async function getTest(testId) {
     }
 }
 
+async function downloadFile(fileUrl) {
+    const options = {
+        url: fileUrl,
+        encoding: null
+    };
+    try {
+        const response = await request.get(options);
+        const base64Value = Buffer.from(response).toString('base64');
+        return base64Value;
+    } catch (err) {
+        const errMsg = 'Error to read file, throw exception: ' + err;
+        console.log(errMsg);
+        const error = new Error(errMsg);
+        error.statusCode = 400;
```

Comment:
I think that 422 might be better what do you think?

422 Unprocessable Entity (WebDAV)
The request was well-formed but was unable to be followed due to semantic errors.

Reply:
fixed